File: BernoulliDist01.py
import numpy as np
from scipy.stats import bernoulli
import matplotlib.pyplot as plt

#Arguments
p=0.3
x = 0,1 #0 - failure, 1 - success
f_x = bernoulli.pmf(x,p) #pdf
f_x = bernoulli.cdf(x,p) #cdf

#Figure
fig = plt.figure()
ax = plt.axes()
ax.plot(x, f_x, 'bo', markersize=8) #'bo' for cirlce markers
#alpha-shading, lw-line weight,b-  blue needles
ax.vlines(x, 0, f_x, colors='b', lw=3, alpha=0.5)

#Random sample of size n
np.random.seed(2022)

n = 1000
rand = bernoulli.rvs(p,size=n)

#Hitogram, alpha - transparency, rwidth - bar width
n, bins, patches = plt.hist(x=rand, bins='auto', color='#0504aa',alpha=0.5, rwidth=0.9)

File: BernoulliDist01.R
#Arguments
x=seq(0, 1, by = 1)

#Pdf and Cdf functions, 1 trial
f_x <- dbinom(x, 1, 0.2) #pdf
f_x <- pbinom(x, 1, 0.2) #cdf

#Needle plot
plot(x,f_x, type="h",main='Proabability plot')
points(x,f_x,pch=16,cex=1,col="blue") #pch - symbol filled circle

#Random sample from normal of size n
set.seed(2022)
n <- 1000
y <-rbinom(10, 1, 0.2)

#Histogram
hist(y, main = "Histogram")
